e2e/test/scenarios/admin/datamodel/reproductions.cy.spec.js

```diff
@@ -107,6 +112,90 @@ describe("issue 21984", () => {
   });
 });
 
+describe("issue 15542", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+
+    cy.wrap(
+      `/admin/datamodel/database/${SAMPLE_DB_ID}/schema/${SAMPLE_DB_SCHEMA_ID}/table/${ORDERS_ID}/field/${ORDERS.PRODUCT_ID}/general`,
+    ).as("ORDERS_PRODUCT_ID_URL");
+    cy.intercept("POST", "/api/field/*/dimension").as("fieldDimensionUpdate");
+  });
+
+  function openOrdersTable() {
+    // Navigate without reloading the page
+    navigationSidebar().findByText("Databases").click({ force: true });
```

Comment:
In this case that is because in some case the sidebar is closed, and it's surprisingly hard to open the sidebar in cypress. I'll add a comment to the code.